sympy/utilities/tests/test_lambdify.py

```diff
@@ -577,3 +577,11 @@ def test_issue_2790():
     assert lambdify((x, (y, z)), x + y)(1, (2, 4)) == 3
     assert lambdify((x, (y, (w, z))), w + x + y + z)(1, (2, (3, 4))) == 10
     assert lambdify(x, x + 1, dummify=False)(1) == 2
+
+def test_incorrect_arg_expr():
+    with raises(ValueError):
+        lambdify(x, "numpy")
+
+def test_incorrect_arg_args():
+    with raises(ValueError):
+        lambdify("numpy", x)
```

Comment:
Currently, this gives the symbol `x` for any single input, `lambdify('numpy', x) -> x`